src/ParsedText.js

```diff
@@ -126,9 +127,7 @@ class ParsedText extends React.Component {
     const { parse, childrenProps, ...remainder } = { ...this.props };
 
     return (
-      <Text ref={(ref) => (this._root = ref)} {...remainder}>
-        {this.getParsedText()}
-      </Text>
+        this.getParsedText()
```

Comment:
This would be a pretty big breaking change to the API -- this Component is intentionally wrapped in a Text component that accepts some style props.